packages/unlock-js/src/subgraph/schema.graphql

```diff
@@ -17,9 +17,12 @@ query allLocks(
     name
     expirationDuration
     tokenAddress
+    symbol
```

Comment:
@searchableguy Note that the symbol here is the ERC721 symbol (`KEY` by default), not the currency symbol... I do think it would be useful to add the `currencyName` and `currencySymbol` too though.